validator-api/src/config/mod.rs

```diff
@@ -224,6 +256,11 @@ impl Config {
         self
     }
 
+    pub fn enabled_rewarding(mut self, enabled: bool) -> Self {
```

Comment:
Maybe
```suggestion
    pub fn with_rewarding_enabled(mut self, enabled: bool) -> Self {
```
to keep it in line with the other functions?

Reply:
Good idea. I will also change `enabled_network_monitor` to follow the same pattern.